jobs/signing/sign-artifacts/umb_producer.py

```diff
@@ -60,6 +60,18 @@
 ART_CONSUMER = 'Consumer.openshift-art-signatory.{env}.VirtualTopic.eng.robosignatory.art.sign'
 
 
+def get_release_tag(release_name, arch):
+    """Determine the quay destination tag where a release image lives, based on the
+    release name and arch (since we can now have multiple arches for each release name)
+    - make sure it includes the arch in the tag to distinguish from any other releases of same name.
+
+    e.g.:
+    (4.2.0-0.nightly-s390x-2019-12-10-202536, s390x) remains 4.2.0-0.nightly-s390x-2019-12-10-202536
+    (4.3.0-0.nightly-2019-12-07-121211, x86_64) becomes 4.3.0-0.nightly-2019-12-07-121211-x86_64
+    """
```

Comment:
well, the inconsistent positions of arch names confuse me a lot. Is this the standard practice or will change?

Reply:
that's not the right place for this question. IDK who determines it, release_controller maybe ??

Reply:
It seems harmless anyway. Just felt confused at the first sight but I am ok to ignore this at the moment.